src/App.js

```diff
@@ -6,14 +6,20 @@ class App extends Component {
   constructor(props) {
     super(props)
     this.state = {
-      wordList: []
+      wordList: JSON.parse(localStorage.getItem("wordList") || "[]")
```

Comment:
🤔I haven't used local storage with React before, but I'm wondering if this is the best place to do this? React recommends loading data from a remote endpoint in [componentDidMount()](https://reactjs.org/docs/react-component.html#componentdidmount), but I'm not sure if they have an opinion on local storage. Might be worth a Googling if you're interested, but this probably also works.

Reply:
Need to read this more but looked like some good documentation https://reactjs.org/docs/state-and-lifecycle.html